src/hooks/useStore.js

```diff
@@ -0,0 +1,105 @@
+import create from 'zustand';
+import { persist } from 'zustand/middleware';
+
+const useStore = create(
+  persist(
+    (set, get) => ({
+      searchInput: '',
+      currentFilter: 'all',
+      photo: '',
+      activities: [],
+      addedFriend: [],
+      addedGroup: [],
+      showBin: true,
+
+      setSearchInput: searchInput => {
+        set({ searchInput: searchInput });
+      },
+      setCurrentFilter: currentFilter => {
+        set({ currentFilter: currentFilter });
+      },
+      setPhoto: photo => {
+        set({ photo: photo });
+      },
+      setActivities: act => {
+        set({ activities: act });
+      },
+      setAddedFriend: fr => {
+        set({ addedFriend: fr });
+      },
+      setAddedGroup: gr => {
+        set({ addedGroup: gr });
+      },
```

Comment:
```suggestion
      setAddedFriend: friend => {
        set({ addedFriend: friend });
      },
      setAddedGroup: group => {
        set({ addedGroup: group });
      },
```

Reply:
oh well yes, I did it for keeping the overview myself, but its better to proper write it, thank you😊